log/tool_logger.go

```diff
@@ -17,16 +17,19 @@ import (
 
 // Tool Logger verbosity constants
 const (
-	Always = iota
+	Error = iota
+	Debug
+	Trace
+	Warn
```

Comment:
I believe the order should be Error, Warn, Info, Debug, Trace. Please let me know what you think.

Reply:
The log levels are ordered in this way so that Debug and Trace, the second and third log levels, respectively, have the values 1 and 2. If we were to employ the ordering you've mentioned, Debug would be 3 and Trace would be 4, which would throw off the mapping from enum to verbosity level.

Reply:
I kinda know why you are doing this, but it's a bit of hacky, because you want to keep the original verbosity level setting but adding two more granular levels in between.